terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/config/PropertySourceConfig.java

```diff
@@ -0,0 +1,15 @@
+package org.terasoluna.gfw.functionaltest.config;

```

Comment:
I did ` license-maven-plugin` by `mvn license:format`, but the results have not changed from now.

But I will try to use ` license-maven-plugin` from the following.